packages/aws-cdk-lib/aws-synthetics/test/canary.test.ts

```diff
@@ -249,7 +249,7 @@ test('Python runtime can be specified', () => {
 
   // WHEN
   new synthetics.Canary(stack, 'Canary', {
-    runtime: synthetics.Runtime.SYNTHETICS_PYTHON_SELENIUM_4_0,
+    runtime: synthetics.Runtime.SYNTHETICS_PYTHON_SELENIUM_5_0,
```

Comment:
I changed this file and replaced some older versions with newer ones. Changes in this file can be reverted if required.